### PR TITLE
Verify Command

### PR DESCRIPTION
**Pull request by COVID-GUH**

### CODE REVIEW

File: src/command/verification/VerifyDiscordCommand.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+/**
+ * Verification Command
+ * @author GUH <contact@covid19.fyi>
+ *
+ * This command will check if the user issuing the command is in the correct channel and needs the verified rank.
+ * If they do, it gives it to them.
+ */
+export class VerifyDiscordCommand extends DiscordCommand {
+  public async execute(): Promise<void> {
+    const verifiedRole = this.message.guild?.roles.cache.filter(role => role.name === VERIFIED_ROLE).first();
```
Comment (freememory) [inline src/command/verification/VerifyDiscordCommand.ts]: Append to roles

File: src/command/verification/VerifyDiscordCommand.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    if (!verifiedRole) throw Error("No verified role found");
+
+    await this.message.member?.roles.set([verifiedRole], "User Verified");
+  }
+
+  public async validate(): Promise<boolean> {
+    if (this.message.channel.id !== VERIFICATION_CHANNEL_ID) return false;
+    if (!this.message.member) return false;
```
Comment (freememory) [inline src/command/verification/VerifyDiscordCommand.ts]: Don't allow verify if user has suspended role

Comment (COVID-GUH) [reply src/command/verification/VerifyDiscordCommand.ts]: I guess this is more protective, but Suspended should not have access to the verify channel.

File: src/command/verification/VerifyDiscordCommand.ts (hunk 1 above)
Comment (COVID-GUH) [reply src/command/verification/VerifyDiscordCommand.ts]: Fixed

File: src/command/verification/VerifyDiscordCommand.ts (hunk 2 above)
Comment (COVID-GUH) [reply src/command/verification/VerifyDiscordCommand.ts]: Fixed

**Pull request merged**